### PR TITLE
Builders programme, economy, and fixes

### PR DESCRIPTION
**Pull request by WRRicht3r**

### CODE REVIEW

File: src/build.md [hunk 1]
```diff
@@ -1,3 +1,71 @@
...
+- Projects that already have users on another network can get XOR
+  grants to pay for porting the projects to the SORA network, as well
+  as development mentorship and advice on how to deploy
+  
+## Decentralized Development at its Best
 
-<!-- TODO: Add info -->
\ No newline at end of file
+SORA is a truly decentralized movement, run by the community and for the community. Everyone is encouraged to actively participate in the development and constant improvement of the network. An example of this is the [RFP development process](https://medium.com/sora-xor/sora-polkaswap-decentralized-development-process-6df207d7518a), a program that allows users to submit requests for new features and subsequently proposals themselves to add new functions and to the network. Some successful proposals include the [XSTUSD stablecoin](https://github.com/sora-xor/rfps/blob/master/open_rfps/XST(USD)%20RFP) and the [Referral Program](https://github.com/sora-xor/rfps/blob/master/open_rfps/Referral%20Program%20Proposal) that will soon be released into production.
```
Comment (bgodlin) [inline src/build.md]: `src/build.md` duplicates the contents from `src/decentralised-development-process.md`

File: src/build.md [hunk 2]
```diff
@@ -1,3 +1,71 @@
...
+
+The SORA Builders Programme is now launching to take the power of decentralization even further, and aid the community to make all their great ideas and suggestions to improve the network a reality. 
+
+## The Two Modalities of the SORA Builders Programme
+
+The SORA Builders Programme is an extension of the [SORA RFP program](https://medium.com/sora-xor/sora-polkaswap-decentralized-development-process-6df207d7518a) (see also [How to RFP](https://medium.com/sora-xor/how-to-rfp-proposing-new-functionalities-to-polkaswap-and-the-sora-network-4af099b15f75)). The SORA Builders Programme branches out into two modalities, **New Project (Incubation)** and **Existing Project (Port).** As the names imply, this programme will support the implementation and development of new projects, as well as porting existing apps onto the SORA network.
```
Comment (bgodlin) [inline src/build.md]: I recommend updating the target links from Medium to Github https://github.com/sora-xor/rfps, as the Medium articles are outdated. Specifically, we have recently transitioned from pull request (PR)-based Request for Proposals (RFPs) to issue-based ones.

File: src/build.md [hunk 3]
```diff
@@ -1,3 +1,71 @@
...
+  team for any support required. Additionally, if your proejct
+  qualifies and you chose to sign up
+  for Social Insurance for [Systemically Important Infrastructure
+  ](social-insurance.md), the governance of your app will be delegated
+  to the SORA Parliament. 
+
+
+## Time to #Buidl
+
+**The fate of the SORA network lies in the hands of its users!** SORA
```
Comment (bgodlin) [inline src/build.md:56]: > I recommend updating the target links from Medium to Github https://github.com/sora-xor/rfps, as the Medium articles are outdated. Specifically, we have recently transitioned from pull request (PR)-based Request for Proposals (RFPs) to issue-based ones.

**Review by bgodlin — CHANGES_REQUESTED** [review]

File: src/sora-economy.md [hunk 4]
```diff
@@ -1,9 +1,149 @@
...
+
+The conclusions were that there is a large amount of empirical evidence that suggests that **it is the change in the allocation of money in an economy that is a necessary and sufficient determinant of growth**. This was shown by Richard Werner in his [disaggregated quantity theory of money](https://eprints.soton.ac.uk/36569/), where he disaggregated Fisher's [equation of exchange](https://en.wikipedia.org/wiki/Equation_of_exchange) (MV = PY; a good way to think about this is also that MV equals nominal GDP viewed from the perspective of buyers, whereas PY equates to nominal GDP viewed from the perspective of producers/sellers) into *real* (*R*) and *financial* (*F*) transactions:
+
+$$
```
Comment (bgodlin) [inline src/sora-economy.md:20]: Formulas are not rendering

File: src/sora-economy.md [hunk 5]
```diff
@@ -1,9 +1,149 @@
...
+
+![](.gitbook/assets/phases.png)
+
+## Towards a New Tomorrow
+
+One of the reasons decentralized P2P economic systems haven't been more widely adopted and lack sophistication is not having access to an elastic supply of capital via a central bank or mutual savings bank. By decentralizing the concept of a central bank, blockchain networks can finally create and allocate capital to fund worthwhile producers as well as needed infrastructure and public goods. With its interoperable technology, SORA can become the decentralized central bank for the world of blockchains, financing the creation of new and exciting applications.
+
+---
+
+## FAQs
```
Comment (bgodlin) [inline src/sora-economy.md]: Should we merge this chapter with the contents in `src/sora-faq.md`?

**Review by bgodlin — CHANGES_REQUESTED** [review]

File: src/sora-economy.md (hunk 5 above)
Comment (WRRicht3r) [reply src/sora-economy.md]: I removed this faq altogether as it is outdated

File: src/build.md [hunk 6]
```diff
@@ -1,3 +1,69 @@
...
+  of the elite SORA Builders Graduates. This doesn't mean that the
+  support provided will be over. You can still reach out to the tech
+  team for any support required. Additionally, if your proejct
+  qualifies and you chose to sign up
+  for Social Insurance for [Systemically Important Infrastructure
+  ](social-insurance.md), the governance of your app will be delegated
+  to the SORA Parliament. 
+
+
+## Time to #Buidl
```
Comment (outoftardis) [inline src/build.md:54]:
```suggestion
## Time to Build
```

typo?

File: src/build.md [hunk 7]
```diff
@@ -1,3 +1,69 @@
...
+
+Both new projects and ports can receive compensation in the form of XOR, as well as development support from the SORA community. Because effectively time and development effort are not free, and in order to uphold the community standards, not everyone who applies will be accepted by the programme.
+
+It is also important to mention that depending on the value added by the project, it may also qualify for [Social Insurance](https://medium.com/sora-xor/social-insurance-for-systematically-important-infrastructure-18a63ef711ca), however, there will be required due diligence and the project will have to prove systematic importance to the community.
+
+It is easy to ask *wot* and *wen* when it comes to development, but *putting your RFP where your complaints are* is not that simple to do. The community has no limitation to what is proposed, however, [*spam](https://youtu.be/duFierM1yDg), CEX requests,* and other off-topic requests will not be considered.
```
Comment (outoftardis) [inline src/build.md]: is `*wot* and *wen*` an intended spelling?

also check preview, this ` [*spam](https://youtu.be/duFierM1yDg), CEX requests,*` does not render correctly 

```suggestion
It is easy to ask *wot* and *wen* when it comes to development, but *putting your RFP where your complaints are* is not that simple to do. The community has no limitation to what is proposed, however, [spam](https://youtu.be/duFierM1yDg), CEX requests, and other off-topic requests will not be considered.
```

you can add a link to where CEX requests are explained, then both spam and CEX requests will stand out in the text around them (as links) without any additional markup

File: src/build.md [hunk 8]
```diff
@@ -1,3 +1,69 @@
 # Build on SORA
+- Developers can receive support to deploy **new and amazing projects on the SORA network**
+- New projects that are developed from scratch can receive financial backing in XOR and project incubation, as well as development support
+- Projects that already have users on another network can get XOR
+  grants to pay for porting the projects to the SORA network, as well
+  as development mentorship and advice on how to deploy
```
Comment (outoftardis) [inline src/build.md]:
```suggestion
- Developers can receive support to deploy **new and amazing projects on the SORA network**.
- New projects that are developed from scratch can receive financial backing in XOR and project incubation, as well as development support.
- Projects that already have users on another network can get XOR
  grants to pay for porting the projects to the SORA network, as well
  as development mentorship and advice on how to deploy.
```

File: src/build.md [hunk 9]
```diff
@@ -1,3 +1,69 @@
...
+  SORA Ecosystem Updates and on SORA social media channels.
+  
+- **Graduation** Once your project is live on SORA, you will be part
+  of the elite SORA Builders Graduates. This doesn't mean that the
+  support provided will be over. You can still reach out to the tech
+  team for any support required. Additionally, if your proejct
+  qualifies and you chose to sign up
+  for Social Insurance for [Systemically Important Infrastructure
+  ](social-insurance.md), the governance of your app will be delegated
+  to the SORA Parliament.
```
Comment (outoftardis) [inline src/build.md]: is there a reason this info is in an unordered list? I also can't tell is the indentation is intentional or not

File: src/build.md [hunk 10]
```diff
@@ -1,3 +1,69 @@
...
+and the **SORA Builders Programme**. Progress from both these
+initiatives will be reported in the fortnightly **Ecosystem Updates**
+and all the submissions, as well as RFP and merged implementations,
+are publicly accessible in the **[SORA RFP Github
+Repository](https://github.com/sora-xor/rfps)** which ensures that
+full transparency and decentralized practices are always upheld.
+
+
+
+For more information, read the articles *[SORA Decentralized Development Process](decentralised-development-process.md)*, [*How to RFP*](rfp.md), and [*Social Insurance for Systematically Important Infrastructure*](social-insurance.md). To stay in the loop with the latest implementations of the SORA network and Polkaswap, make sure to follow the Ecosystem updates, and finally, to get a taste of what is to come, watch the **[SORA Development Public Request for Proposal (RFP) Process Github Repo](https://github.com/sora-xor/rfps)**!
```
Comment (outoftardis) [inline src/build.md:71]:
```suggestion
For more information, read the articles *[SORA Decentralized Development Process](decentralised-development-process.md)*, [*How to RFP*](rfp.md), and [*Social Insurance for Systematically Important Infrastructure*](social-insurance.md). To stay in the loop with the latest implementations of the SORA network and Polkaswap, make sure to follow the Ecosystem updates. And finally, to get a taste of what is to come, watch the **[SORA Development Public Request for Proposal (RFP) Process Github Repo](https://github.com/sora-xor/rfps)**!
```

File: src/build.md (hunk 6 above)
Comment (WRRicht3r) [reply src/build.md:54]: LOL it's a crypto term, done on purpose :P

Comment (outoftardis) [reply src/build.md:54]: haha ok my bad

File: src/sora-economy.md (hunk 4 above)
Comment (bgodlin) [reply src/sora-economy.md:20]: The dependencies will be cherry-picked from https://github.com/sora-xor/sora-docs/pull/42

File: src/build.md [hunk 11]
```diff
@@ -1,3 +1,70 @@
...
+
+## The Two Modalities of the SORA Builders Programme
+
+The SORA Builders Programme is an extension of the [SORA RFP program](rfp.md). The SORA Builders Programme branches out into two modalities, **New Project (Incubation)** and **Existing Project (Port).** As the names imply, this programme will support the implementation and development of new projects, as well as porting existing apps onto the SORA network. 
+
+Both new projects and ports can receive compensation in the form of XOR, as well as development support from the SORA community. Because effectively time and development effort are not free, and in order to uphold the community standards, not everyone who applies will be accepted by the programme.
+
+It is also important to mention that depending on the value added by the project, it may also qualify for [Social Insurance](https://medium.com/sora-xor/social-insurance-for-systematically-important-infrastructure-18a63ef711ca), however, there will be required due diligence and the project will have to prove systematic importance to the community.
```
Comment (bgodlin) [inline src/build.md]: Should we replace a medium article with https://github.com/sora-xor/sora-docs/blob/develop/src/social-insurance.md?

**Review by bgodlin — CHANGES_REQUESTED** [review]

File: src/sora-economy.md (hunk 4 above)
Comment (bgodlin) [reply src/sora-economy.md:20]: https://github.com/sora-xor/sora-docs/issues/70

**Review by bgodlin — APPROVED** [review]

File: src/build.md [hunk 12]
```diff
@@ -1,3 +1,71 @@
...
+4- **Co-Marketing**: Part of the support provided from the SORA
+  Builder's Programme includes amplification of the project in the
+  SORA Ecosystem Updates and on SORA social media channels.
+  
+5- **Graduation**: Once your project is live on SORA, you will be part
+  of the elite SORA Builders Graduates. This doesn't mean that the
+  support provided will be over. You can still reach out to the tech
+  team for any support required. 
+  
+Additionally, if your proejct qualifies and you chose to sign up for Social Insurance for [Systemically Important Infrastructure
```
Comment (outoftardis) [inline src/build.md]:
```suggestion
Additionally, if your project qualifies and you chose to sign up for Social Insurance for [Systemically Important Infrastructure
```

File: src/sora-economy.md [hunk 13]
```diff
@@ -1,9 +1,124 @@
...
+  - - link
+    - rel: stylesheet
+      href: https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.5.1/katex.min.css
+---
 
-<!-- TODO: Add info -->
+# SORA: The New Economic Order
+- SORA v2 tokenomic model is proposed that uses a token bonding curve with an increasing price function. This means that as the token supply increases, the price also increases.
+- Reserves held by the token bonding curve will provide liquidity for XOR.
+- A new, staking reward token, tentatively called VAL is proposed.
```
Comment (outoftardis) [inline src/sora-economy.md:4]:
```suggestion
- A new, staking reward token called VAL is proposed.
```

File: src/sora-economy.md [hunk 14]
```diff
@@ -1,9 +1,124 @@
...
 
-<!-- TODO: Add info -->
+To realize this system, we are proposing SORA: The New Economic Order. This is a profound and significant upgrade to the SORA Network and the SORA tokenomics. The [upgrade was voted on](https://medium.com/sora-xor/sora-new-economic-order-tokenomics-referendum-31581e8b649) and [ratified by the SORA community](https://medium.com/sora-xor/sora-v2-implementation-1febd3260b87). We feel that this is a system that can add value to humanity and we strongly support its adoption.
+
+## SORAnomics
+
+The key thesis of the SORA economic system is that **for cryptoeconomic systems to be able to compete with contemporary, centralized economic systems, there needs to be a rational economic model**. To create a rational economic model, we did a survey of various economic theories, starting our journey in 2017. Over the course of this time we also employed an economist ([Yokei Yamaguchi](https://www.vollgeld-initiative.ch/fa/img/Vertiefung_deutsch/P2P-PM-System_WP__Japan_Futures_Research_Center.pdf)) to work with us in 2018, and he helped us with several key parts of the model.
```
Comment (outoftardis) [inline src/sora-economy.md:16]:
```suggestion
The key thesis of the SORA economic system is that **for cryptoeconomic systems to be able to compete with contemporary centralized economic systems, there needs to be a rational economic model**. To create a rational economic model, we did a survey of various economic theories, starting our journey in 2017. Over the course of this time we also employed an economist ([Yokei Yamaguchi](https://www.vollgeld-initiative.ch/fa/img/Vertiefung_deutsch/P2P-PM-System_WP__Japan_Futures_Research_Center.pdf)) to work with us in 2018, and he helped us with several key parts of the model.
```

File: src/sora-economy.md [hunk 15]
```diff
@@ -1,9 +1,124 @@
...
...goods and services that consume this new purchasing power.** However, for financial transactions, there are no new goods or services being created with the new money put into circulation, so this causes a rise in asset prices to compensate (which is not what you want in your economic system). Additionally, it is also important to understand the importance of expanding M for the production of new goods and services. As [Shimomura wrote about extensively](https://www.jstage.jst.go.jp/article/economics1950/11/3-4/11_3-4_1/_article/-char/ja/) and Kurihara distilled to a simple, [linear programming model](https://books.google.co.jp/books?id=RVtGAQAAQBAJ&pg=PA60&lpg=PA60&dq=Japan%E2%80%99s+Credit-Supported+Growth+and+Linear+Programming&source=bl&ots=dNNHw-QyMh&sig=ACfU3U0gv9WNMZbWJBlfDf7AL5PY7aZzTQ&hl=ja&sa=X&ved=2ahUKEwjV8rf2qfHoAhUQx4sBHf2cBaEQ6AEwAHoECAwQKw#v=onepage&q=Japan%E2%80%99s%20Credit-Supported%20Growth%20and%20Linear%20Programming&f=false), expanding the quantity of money for new goods and services expands the economic output because latent resources (especially human resources) are mobilized, and existing resources are upgraded by the creation of new capital for production.
```
Comment (outoftardis) [inline src/sora-economy.md]: I'd break the paragraph
1) before " However, for financial transactions"
2) before "Additionally, it is also important"

File: src/sora-economy.md [hunk 16]
```diff
@@ -1,9 +1,124 @@
...
+
+- **Autonomous Management of Token Supply to Match Demand (Elastic Supply)** - The token bonding curve introduces and removes XOR from circulation to meet the demand of the market. This ensures sustainable economic growth and price stability of the token economy, as the system can adapt to the changing needs.
+- **Deterministic Price Calculation** - The buy and sell prices of tokens in the primary market increase and decrease with changes in the supply of token. It is therefore **NOT** a fixed exchange rate system. The buy-price function starting price will be determined closer to the v2 launch and increase by $0.01 for each 5000 XOR sold, even if this is not final and may change before launch.
+- **Continuous Price Dynamics** - Given that the price of token N is inferior to token N+1 and superior to token N-1, calculating the number of tokens minted for a given amount of buy or sell orders requires integral calculus.
+- **Deep and Immediate Liquidity** - The bonding curve contract is the counterpart of the transaction and always holds enough buyback reserves (if investors want to sell tokens back at the current sell-price).
```
Comment (outoftardis) [inline src/sora-economy.md:96]:
```suggestion
- **Autonomous Management of Token Supply to Match Demand (Elastic Supply)**: The token bonding curve introduces and removes XOR from circulation to meet the demand of the market. This ensures sustainable economic growth and price stability of the token economy, as the system can adapt to the changing needs.
- **Deterministic Price Calculation**: The buy and sell prices of tokens in the primary market increase and decrease with changes in the supply of token. It is therefore **NOT** a fixed exchange rate system. The buy-price function starting price will be determined closer to the v2 launch and increase by $0.01 for each 5000 XOR sold, even if this is not final and may change before launch.
- **Continuous Price Dynamics**: Given that the price of token N is inferior to token N+1 and superior to token N-1, calculating the number of tokens minted for a given amount of buy or sell orders requires integral calculus.
- **Deep and Immediate Liquidity**: The bonding curve contract is the counterpart of the transaction and always holds enough buyback reserves (if investors want to sell tokens back at the current sell-price).
```

File: src/sora-economy.md [hunk 17]
```diff
@@ -1,9 +1,124 @@
...
+
+Features specific to the SORA token bonding curve-based economic design:
+
+- **Separation of primary & secondary markets:** Primary market - issuance/withdrawal, Secondary market - exchange rates, arbitrage trading.
```
Comment (outoftardis) [inline src/sora-economy.md:100]:
```suggestion
- **Separation of primary & secondary markets:** Primary market: issuance/withdrawal, Secondary market: exchange rates, arbitrage trading.
```

File: src/sora-economy.md (hunk 15 above)
Comment (WRRicht3r) [reply src/sora-economy.md]: Added breaks

**Review by outoftardis — APPROVED** [review]

**Pull request merged**